app/code/Magento/Sales/Model/Order.php

```diff
@@ -3,6 +3,8 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```

Comment:
This is correct but there is a big risk. For example, "public function getCustomerName ()" will be fail. I think this change doesn't relate to this PR.